src/Web3.elm

```diff
@@ -178,17 +179,31 @@ toChecksumAddress (Address address) =
         }
 
 
-
--- fromWei : EthUnit -> BigInt -> BigInt
--- toWei : EthUnit -> BigInt -> BigInt
--- CORE
-
-
 toWei : EthDenomination -> BigInt -> BigInt
 toWei unit amount =
     mul amount (getValueOfUnit unit)
 
 
+fromWei : EthDenomination -> BigInt -> Float
+fromWei unit amount =
+    let
+        unitDegree =
+            getValueOfUnit unit
+
+        bigIntToFloat value =
```

Comment:
I'm open to suggestions here, it would be better if this was just `bigIntToFloat = ...` but I was too n00b to work out the piping then.